pkg/infra/tower.go

```diff
@@ -46,9 +50,9 @@ func NewTower(maxPlatFood float64, agentCount,
 
 func (t *Tower) Tick() {
 	//logs
-	t.Log("", Fields{"Curr Tower Tick": t.dayInfo.CurrTick})
-	t.Log("Reporting platform status", Fields{"food_left": t.currPlatFood, "floor": t.currPlatFloor})
-
+	t.mx.RLock()
+	t.TowerStateLog(" end of day")
+	t.mx.RUnlock()
```

Comment:
I'm not sure we need this mutex - I don't think Tick() is ever called by multiple threads?

Reply:
Truuuuee